src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid transparent;
+  border-radius: 4px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__content {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    align-self: baseline;
+    color: #616070;
+    margin: 4px 0 16px;
+  }
+
+  &__price-description {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 0 16px;
+  }
+
+  &__price {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+
+    &-number {
+      font-weight: 700;
+      line-height: 18px;
+      font-size: 16px;
+      margin-left: 86px;
+    }
+  }
+
+  .btn {
+    font-family: inherit;
+    line-height: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 14px;
+    font-weight: 700;
+    color: #fff;
+    background-color: #00acdc;
+    text-transform: uppercase;
+  }
+
+  &__btn {
+    border-radius: 5px;
+    border: none;
+    width: 166px;
+    height: 40px;
```

Comment:
no needs to hardcode height and width.
width should be 100%
instead of hardcoded height it's better to use padding